src/main/locales/en/translation.json

```diff
@@ -1,6 +1,84 @@
 {
+  "template": {
+    "serviceName": "Apply to adopt a child",
+    "phaseBanner": {
+      "phase": "Alpha",
+      "content": "This is a new service - your <a href='#'> feedback </a> will help us to improve it."
+    },
+    "links": {
+      "profile": "Profile",
+      "signOut": "Sign out"
+    }
+  },
   "home": {
-    "title": "Hello world!"
+    "heading": "Hello world!"
+  },
+  "error": {
+    "heading": "Something went wrong"
+  },
+  "pageNotFound": {
+    "heading": "Page not found"
   },
-  "single": "single works"
+  "taskList": {
```

Comment:
Would it be beneficial to split this file into multiple ones? I am afraid it's going to be annoying to collaborate on it.

Reply:
Yeah but's let's do this as a tech task.

Reply:
I would probably extract the ones you have added

Reply:
Extracted to two different namespaces for now. `common` to hold all common translations e.g button text, back link etc. Feature model `taskList` has then be added to house translations specific to a page / feature.